build.yaml

```diff
@@ -774,7 +775,6 @@ run_trial_cli: &run_trial_cli [
   *setup_venv,
   *setup_flocker_modules,
   *setup_coverage,
-  *setup_aws_env_vars,
```

Comment:
They're the variables required to run the `IBlockDeviceAPI` functional tests on AWS nodes.
I'm pretty sure they have been copy pasted all over the place without them actually being required.